language/stdlib/modules/libra_system.mvir

```diff
@@ -24,7 +24,18 @@ module LibraSystem {
 
     resource BlockMetadata {
       // Height of the current block
+      // TODO: Should we keep the height?
       height: u64,
+
+      // current real world time provided by consensus.
+      timestamp: u64,
+
+      // hash of the previous block
+      id: bytearray,
```

Comment:
Worth reflecting the "previous" part in the name,

Reply:
I've updated the comment in the new fixup